policy/modules/kernel/domain.te

```diff
@@ -245,6 +245,7 @@ optional_policy(`
 
 # Use bpf tools
 allow unconfined_domain_type self:bpf { map_create map_read map_write prog_load prog_run };
+allow unconfined_domain_type self:capability2 perfmon;
```

Comment:
This should be unnecessary thanks to:
https://github.com/fedora-selinux/selinux-policy/blob/46d834991b9d07400b9feb8aee2d85e40cfe95d7/policy/modules/system/unconfined.if#L23

Reply:
Fixed.